lib/checks/aria/fallbackrole.js

```diff
@@ -0,0 +1 @@
+return virtualNode.attr('role').split(' ').length > 1;
```

Comment:
This should probably use `axe.utils.tokenList`. Otherwise `role="  button  "` would fail the rule.